django/middleware/common.py

```diff
@@ -161,8 +161,14 @@ def is_ignorable_request(self, request, uri, domain, referer):
         """
         Returns True if the given request *shouldn't* notify the site managers.
         """
-        # '?' in referer is identified as search engine source
+        # Ignore three specific behaviours:
+        # - empty referer
+        # - '?' in referer is identified as search engine source
+        # - referer equal to current URL (with or without domain) as a malicious bot (#25302)
```

Comment:
We try to omit ticket references in code (not tests) unless the issue is very obscure -- I don't think it's needed here. Please wrap comments/docstrings at 79 chars.